src/app/shared/helpers/cast-movie.mapper.ts

```diff
@@ -0,0 +1,16 @@
+import { MediaType } from '../enums/media-types';
+import {
+  CastCard,
+  MovieCastDTO
+} from '../interfaces/cast';
+
+export function castMovieMapper(obj: MovieCastDTO): CastCard {
+  return {
+    id: obj.id ?? 0,
+    name: obj.original_title ?? '',
+    date: obj.release_date ?? new Date(),
```

Comment:
Do you get Date object from backend? Please, check the dev tools. I'm not sure you do.

In case, I'm right, please use map to Date object.
```
data: obj.release_date ? new Date(obj.release_date) : new Date();
```

It's related to all backend dates

Reply:
Done